packages/flakybot/src/flakybot.ts

```diff
@@ -150,7 +150,16 @@ export function flakybot(app: Probot) {
   app.on('schedule.repository' as any, async context => {
     const owner = context.payload.organization.login;
     const repo = context.payload.repository.name;
-    await syncLabels(context.octokit, owner, repo, FLAKYBOT_LABELS);
+    let octokit: Octokit;
+    if (context.payload.installation && context.payload.installation.id) {
```

Comment:
Option for conciseness:
```suggestion
    if (context.payload.installation?.id) {
```